.github/CODEOWNERS

```diff
@@ -1003,6 +1003,9 @@
 # ServiceLabel: %Network - Mobile %Mgmt
 # ServiceOwners:                                                   @ArthurMa1978
 
+# ServiceLabel: %NetworkCloud %Mgmt
+# ServiceOwners:                                                   @Azure/azure-sdk-write-networkcloud
+
```

Comment:
Correct. Since we're going with `Operator Nexus - Network Cloud`, we should remove/change all parts of CODEOWNERS that used the `NetworkCloud` label. 

@jsquire, would you also want to change the file structure for where the library source is kept?

Reply:
> Correct. Since we're going with `Operator Nexus - Network Cloud`, we should remove/change all parts of CODEOWNERS that used the `NetworkCloud` label.
> 
> @jsquire, would you also want to change the file structure for where the library source is kept?

That we can't really do, as it is automated and based on the RP name.